helm/ph-ee-engine/values.yaml

```diff
@@ -71,7 +94,7 @@ zeebe-cluster-helm:
     - name: init-ph-ee-kafka-exporter
       image: busybox:1.28
       command: ['/bin/sh', '-c']
-      args: ['wget -O /exporters/ph-ee-kafka-exporter.jar "https://paymenthub-ee-dev.s3.us-east-2.amazonaws.com/jars/exporter-1.0.0-SNAPSHOT.jar"; ls -al /exporters/']
+      args: ['wget -O /exporters/ph-ee-kafka-exporter.jar "https://paymenthub-ee-dev.s3.us-east-2.amazonaws.com/jars/exporter-1.1.0-SNAPSHOT.jar"; ls -al /exporters/']
```

Comment:
Can't use S3 for this. Don't use versioning in the gradle build - let github tags and pipelines handle that. 

The pipeline is failing as exporter 1.1.0 is still producting exporter-1.0.0 jar because of version being in build.gradle. - http://ec2-3-143-126-230.us-east-2.compute.amazonaws.com:8081/view/PHEE%20Releases/job/PHEE%20Exporter%20Release/configure

Reply:
what will be the url?

Reply:
https://fynarfin.io/images/exporter-1.1.0-SNAPSHOT.jar